android/templates/module/generated/build.gradle

```diff
@@ -120,20 +120,34 @@ android {
 			}
 		}
 	}
-
 	compileOptions {
 		sourceCompatibility JavaVersion.VERSION_1_8
 		targetCompatibility JavaVersion.VERSION_1_8
 	}
+	kotlinOptions {
+		jvmTarget = '1.8'
```

Comment:
Use the same `JavaVersion.VERSION_1_8` reference? as used in the compile options above?

Reply:
I just confirmed that `JavaVersion.VERSION_1_8` returns `1.8`.
So, yeah, I'll go ahead and do that.
(The above was just blindly following Google's own examples.)